docs/user/exams/students_guide.rst

```diff
@@ -217,9 +220,6 @@ Participating in Programming Exercises
     - You can add new files and directories using the |add_file|  and |add_folder| buttons.
     - You also have the ability to rename |rename| and delete |delete| files and folders, therefore **caution** is advised.
 
-        .. note::
-            If you accidentally delete or remove a file, you can use |refresh_files|, to load the last saved version from the server.
```

Comment:
Why is this removed?

Reply:
This doesn't work anymore ^^' since the programming exercises now get autosaved immediately when a change happens, reloading the files after you deleted a file doesn't bring the file back anymore.

Reply:
I just tested this and you are right - It just undoes all uncommitted changes in files that were not deleted. As long as this issue is not resolved, we should indeed delete this wrong information from the doc.
Thanks for the clarification!